### ISSUE TITLE
Support setting environment variables per connection

### ISSUE THREAD

**Issue by dveloz2** [dbeaver/dbeaver#36059]
### Description

I'm trying to run a simple  `export` command to set an environment variable before connection is made. 
I'm getting the following error: 

```
Can't start process
  Cannot run program "export": error=2, No such file or directory
  Cannot run program "export": error=2, No such file or directory
    error=2, No such file or directory
    error=2, No such file or directory
```

### DBeaver Version

Community Edition 24.2.3.202410201725

### Operating System

macOs Sonoma 14.1.1

### Database and driver

I tested with Athena driver and DuckDB driver (latest versions).

### Steps to reproduce

1. Create a connection with any driver.
2. Add a shell command "Before Connect" which executes an `export` command
3. Try to test the connection.
4. See the error.

**Comment by dveloz2** [comment]
<img width="1714" alt="Screenshot 2024-10-24 at 4 13 36 p m" src="https://github.com/user-attachments/assets/ff2bb915-3019-42a0-a284-2e9a566b28bd">

**Comment by ShadelessFox** [comment]
Hello @dveloz2,

The `export` command is a bash command. It cannot be used to set environment variables when spawning a new process.

What are you trying to do, exactly? Just change environment variables without actually running anything? If it were to work, it would set variables just for the spawned process, and they wouldn't propagate to the environment variables of the driver.

If you want to pass environment variables to the driver, consider passing `-D<varname>=<varvalue>` after the `-vmargs` option inside the `dbeaver.ini` file or as command-line arguments.

**Comment by dveloz2** [comment]
Thanks!

Yes I need to pass env variables to the driver's vm. Having different values for different Database Connections.

If i modify the `dbeaver.ini` then all my Database Connections will have the same env vars wouldn't they? Is there a way to apply them selectively?

**Comment by dveloz2** [comment]
Something like the `VM Environments` field available in DataGrip

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/5c252750-56f9-41f5-9667-6b0aaebfd880">

**Comment by ShadelessFox** [comment]
> If i modify the dbeaver.ini then all my Database Connections will have the same env vars wouldn't they? Is there a way to apply them selectively?

Yes, unfortunately. As of now, DBeaver does not allow specifying environment variables per connection.

I will convert the ticket to a feature request, then.
